### ISSUE TITLE
Set precision to more than 100

### ISSUE THREAD

**Issue by frostblooded** [akubera/bigdecimal-rs#62]
If i understand correctly, the maximum allowed precision is 100 even if I try to use with_prec. This is very low number. I am trying to make a program that calculates Pi. Isn't there a way to increase the precision to something like 1,000,000?

**Comment by akubera** [comment]
Well... you can clone and change the hard-coded value until the `feature/simple-context` branch is finally merged.

I think there are a few places that fix that precision to 100, like `let max_precision = 100;`

**Comment by akubera** [comment]
If you want something a little less disappointing, maybe the [rug crate](https://crates.io/crates/rug) would be better in the meantime. It wraps the standard [GNU MPFR](https://www.mpfr.org/) library, so it will be as efficient and accurate as you want.

**Comment by frostblooded** [comment]
Nah, mate. I never meant to imply this crate is disappointing. I just wondered if I have missed something. Keep up the good work! :)

**Comment by WyseNynja** [comment]
Is there an issue or PR tracking the `simple-context` branch? Anything blocking the merge? I have a similar problem as OP and I think this would work for me. I'll look at rug though.
